CondCore/Utilities/scripts/uploadConditions_v2.py

```diff
@@ -0,0 +1,473 @@
+
+
+def getInputRepeat(prompt = ''):
+    '''Like raw_input() but repeats if nothing is provided and automatic strip().
+    '''
+
+    while True:
+        answer = raw_input(prompt)
+        if answer:
+            return answer.strip()
+
+        print('You need to provide a value.')
+
+def runWizard(basename, dataFilename, metadataFilename):
+    while True:
+        print('''\nWizard for metadata for %s
+
+I will ask you some questions to fill the metadata file. For some of the questions there are defaults between square brackets (i.e. []), leave empty (i.e. hit Enter) to use them.''' % basename)
+
+        # Try to get the available inputTags
+        try:
+            dataConnection = sqlite3.connect(dataFilename)
+            dataCursor = dataConnection.cursor()
+            dataCursor.execute('select name from sqlite_master where type == "table"')
+            tables = set(zip(*dataCursor.fetchall())[0])
+
+            # only conddb V2 supported...
+            if 'TAG' in tables:
+                dataCursor.execute('select NAME from TAG')
+            # In any other case, do not try to get the inputTags
+            else:
+                raise Exception()
+
+            inputTags = dataCursor.fetchall()
+            if len(inputTags) == 0:
+                raise Exception()
+            inputTags = list(zip(*inputTags))[0]
+
+        except Exception:
+            inputTags = []
+
+        if len(inputTags) == 0:
+            print('\nI could not find any input tag in your data file, but you can still specify one manually.')
+
+            inputTag = getInputRepeat(
+                '\nWhich is the input tag (i.e. the tag to be read from the SQLite data file)?\ne.g. BeamSpotObject_ByRun\ninputTag: ')
+
+        else:
+            print('\nI found the following input tags in your SQLite data file:')
+            for (index, inputTag) in enumerate(inputTags):
+                print('   %s) %s' % (index, inputTag))
+
+            inputTag = getInputChoose(inputTags, '0',
+                                      '\nWhich is the input tag (i.e. the tag to be read from the SQLite data file)?\ne.g. 0 (you select the first in the list)\ninputTag [0]: ')
+
+        destinationDatabase = ''
+        ntry = 0
+        while ( destinationDatabase != 'oracle://cms_orcon_prod/CMS_CONDITIONS' and destinationDatabase != 'oracle://cms_orcoff_prep/CMS_CONDITIONS' ): 
+            if ntry==0:
+                inputMessage = \
+                '\nWhich is the destination database where the tags should be exported? \nPossible choices: oracle://cms_orcon_prod/CMS_CONDITIONS (for prod) or oracle://cms_orcoff_prep/CMS_CONDITIONS (for prep) \ndestinationDatabase: '
+            elif ntry==1:
+                inputMessage = \
+                '\nPlease choose one of the two valid destinations: \noracle://cms_orcon_prod/CMS_CONDITIONS (for prod) or oracle://cms_orcoff_prep/CMS_CONDITIONS (for prep) \
```

Comment:
Would it be possible to simplify this and ask the user to choose between "Prep" or "Prod" instead of the full sting? Or even 0 and 1, the same way when we choose the tags? That would make the user's life a bit more convenient. Thanks!

Reply:
Thank you for the suggestion, it is a great idea.
I have just added that functionality and updated the PR. The suggestions to the user are oradev and orapro to keep consistency with the conddb tools, however it will also resolve to the proper destination if you input "prep" or "prod".

Reply:
Awesome, thanks!